llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -749,7 +750,7 @@ PassBuilder::buildInlinerPipeline(OptimizationLevel Level,
 
   ModuleInlinerWrapperPass MIWP(IP, PerformMandatoryInliningsFirst,
                                 InlineContext{Phase, InlinePass::CGSCCInliner},
-                                UseInlineAdvisor, MaxDevirtIterations);
+                                UseInlineAdvisor, PTO.MaxDevirtIterations == -1 ? MaxDevirtIterations : PTO.MaxDevirtIterations);
```

Comment:
There is another usage of the global `MaxDevirtIterations` which is not adjusted in this PR: https://github.com/Kotlin/llvm-project/blob/9727c77d58ac920a4158d08c15659470e52ddda4/llvm/lib/Analysis/CallGraphSCCPass.cpp#L543
I would guess that it is a part of the legacy pass manager, and thus is irrelevant. Could you please check that? I can think of a simple sanity check -- at least make sure that this patch changes what we need: put `1` here, recompile LLVM and run `$llvm/bin/opt "-passes=default<O3>" --print-pipeline-passes /dev/null`. It should have `devirt<1>` instead of `devirt<4>`.

Reply:
I've checked (put `1`) and `opt` outputs `devirt<1>`.